sql/moz-fx-data-shared-prod/firefox_accounts/docker_fxa_customs_sanitized/view.sql

```diff
@@ -2,6 +2,105 @@ CREATE OR REPLACE VIEW
   `moz-fx-data-shared-prod.firefox_accounts.docker_fxa_customs_sanitized`
 AS
 SELECT
-  *
+  `date`,
+  logName,
+  (
+    SELECT AS STRUCT
+      resource.type,
+      (
+        SELECT AS STRUCT
+          resource.labels.instance_id,
+          resource.labels.zone,
+          resource.labels.project_id,
+          CAST(NULL AS STRING) AS cluster_name,
+          CAST(NULL AS STRING) AS namespace_name,
+          CAST(NULL AS STRING) AS container_name,
+          CAST(NULL AS STRING) AS pod_name,
+          CAST(NULL AS STRING) AS location
+      ) AS labels
+  ) AS resource,
+  textPayload,
+  (
+    SELECT AS STRUCT
+      jsonPayload.block,
+      jsonPayload.op,
+      jsonPayload.action
+  ) AS jsonPayload,
```

Comment:
There are many fields in `jsonPayload`. How did you determine that these three are the only ones that should be included?

Reply:
I asked Wil on Slack. 
<img width="798" alt="image" src="https://github.com/mozilla/bigquery-etl/assets/42538694/87d0ce53-2aec-4438-b149-de3a50ab8d6c">

Reply:
Also, since this union is at the view level we can always easily add more fields to the view in case more are required.